cert/cert_v1.go

```diff
@@ -317,6 +317,56 @@ func (c *certificateV1) fromTBSCertificate(t *TBSCertificate) error {
 		issuer:         t.issuer,
 	}
 
+	return c.validate()
+}
+
+func (c *certificateV1) validate() error {
+	//TODO: is empty name allowed?
```

Comment:
empty-string is probably allowed, but gross

null should not be allowed, if it's even possible